woocommerce/woocommerce
# Add SlotFill for Task List Footer

### All Submissions:






### Changes proposed in this Pull Request:



A SlotFill extensibility point is added below the Task List to allow customization of this region.

Closes https://github.com/woocommerce/woocommerce/issues/36466 .

<img width="707" alt="Screen Shot 2023-01-20 at 5 22 15 pm" src="https://user-images.githubusercontent.com/9312929/213726027-f4745447-2151-4f11-b1ef-d921acd506f9.png">







### How to test the changes in this Pull Request:



1. Checkout this branch first
1. Merge `add/example-tasklist-footer-fill` on top or cherry pick the relevant [commit](https://github.com/woocommerce/woocommerce/commit/914106d8b4be75748323dadc19fb2aa027b362ed) for the example fill
1. Observe the example fill should look like the above screencap



### Other information:




### FOR PR REVIEWER ONLY:
